### PR TITLE
Oneinch updates 3

### PR DESCRIPTION
**Pull request by max-morrow**
- cleaned exchange contracts
- created 1inch LOP & LOP trades spellbooks

### CODE REVIEW

**Comment by grkhr** [comment]
+ filtered broken tx in dex_aggregator.trades

**Comment by couralex6** [comment]
Hey @max-morrow, we just merged a change that affects the way we define aliases on spells. We now use `alias = 'some-alias'` instead of `alias = alias('some-alias')`.

You'll have to update your PR for the CI to run properly

File: models/oneinch/arbitrum/oneinch_arbitrum_schema.yml [hunk 1]
```diff
@@ -12,13 +12,13 @@ models:
         oneinch arbitrum calls & transfers
 
     columns:
-      - &blockchain
+      - &blockchain_not_null
```
Comment (jeff-dude) [inline models/oneinch/arbitrum/oneinch_arbitrum_schema.yml:15]: what is the intention for adding `_not_null` to the column name itself in the schema files?

typically, we see the columns in schema files match the models themselves, for documentation purposes. curious if you have a strong opinion on this approach?

**Review by jeff-dude — COMMENTED** [review]
just one minor comment/open question below.

yet another clean & innovate spell from the 1inch team 🔥  thank you for this! i'm learning plenty myself reviewing these. we'll need to tag top contributors to share these approaches.

File: models/oneinch/arbitrum/oneinch_arbitrum_schema.yml (hunk 1 above)
Comment (grkhr) [reply models/oneinch/arbitrum/oneinch_arbitrum_schema.yml:15]: @jeff-dude `_not_null` columns contains "not null" test always. it's convenient in huge yml files where column can be null in one schema and can't be null in another.

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**